coderd/deploymentconfig.go

```diff
@@ -5,20 +5,33 @@ import (
 
 	"github.com/coder/coder/coderd/httpapi"
 	"github.com/coder/coder/coderd/rbac"
+	"github.com/coder/coder/codersdk"
 )
 
 // @Summary Get deployment config
 // @ID get-deployment-config
 // @Security CoderSessionToken
 // @Produce json
 // @Tags General
-// @Success 200 {object} codersdk.DeploymentConfig
+// @Success 200 {object} codersdk.DeploymentConfigAndOptions
 // @Router /config/deployment [get]
 func (api *API) deploymentConfig(rw http.ResponseWriter, r *http.Request) {
 	if !api.Authorize(r, rbac.ActionRead, rbac.ResourceDeploymentConfig) {
 		httpapi.Forbidden(rw)
 		return
 	}
 
-	httpapi.Write(r.Context(), rw, http.StatusOK, api.DeploymentConfig)
+	scrubbedConfig, err := api.DeploymentConfig.Scrub()
+	if err != nil {
+		httpapi.InternalServerError(rw, err)
+		return
+	}
+
+	httpapi.Write(
+		r.Context(), rw, http.StatusOK,
+		codersdk.DeploymentConfigAndOptions{
```

Comment:
Seems like these are more like values and usage. Config and options sound too similar, it's difficult to differentiate. Not sure on the naming I suggested, just an option.